app/models/finance/schedule.rb

```diff
@@ -7,4 +7,15 @@ class Finance::Schedule < ApplicationRecord
   def self.default
     find_by(name: "ECF September standard 2021")
   end
+
+  def milestone_for_declaration_type
+    {
+      "started" => milestones[0],
```

Comment:
Why do milestones not have a `declaration_type`?

Reply:
Hmm, when I was adding them I was not convinced they would have a single declaration type.

Now I know for sure that the real-life milestones won't have a single declaration type. Buuut, the milestones we use for validation should. I think. So I can probably add them.

Reply:
Should be in now. I made a migration to populate it on existing milestones, because there are like 20 of them total, and splitting this into multiple PRs and deploys felt like an overkill. If you feel strongly about it I can re-visit.